Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/UI/Plan.hs

```diff
@@ -607,9 +621,10 @@ createMandateInvoiceAndOrder serviceName driverId merchantId merchantOpCityId pl
             vehicleNumber = Nothing,
             badDebtRecoveryDate = Nothing,
             merchantOperatingCityId = merchantOpCityId,
-            serviceName
+            serviceName,
+            currency
           }
-    calculateDues driverFees = sum $ map (\dueInvoice -> roundToHalf (fromIntegral dueInvoice.govtCharges + dueInvoice.platformFee.fee + dueInvoice.platformFee.cgst + dueInvoice.platformFee.sgst)) driverFees
+    calculateDues driverFees = sum $ map (\dueInvoice -> roundToHalf (dueInvoice.govtCharges + dueInvoice.platformFee.fee + dueInvoice.platformFee.cgst + dueInvoice.platformFee.sgst)) driverFees
```

Comment:
do we need roundToHalf now, when we use different currencies?

Reply:
Yeah we should not do it in case of dollar. Maybe you can handle it on basis of currency :(